theme/templates/pages/homepage.html

```diff
@@ -87,111 +90,165 @@ <h4>Give your funders confidence that your data will be available in perpetuity
+                </div>
+
+                <div class="feature col-xs-12 col-sm-6">
+                    <div class="feature-icon"><i class="fa fa-window-maximize fa-5x" aria-hidden="true"></i></div>
+                    <p>Tools for collaboration, enabling you to manage access and more easily work together</p>
+                </div>
+
+                <div class="feature col-xs-12 col-sm-6">
+                    <div class="feature-icon"><i class="fa fa-graduation-cap fa-5x" aria-hidden="true"></i></div>
+                    <p>Tools for engaging students in active learning</p>
+                </div>
+
+                <div class="feature col-xs-12 col-sm-6">
+                    <div class="feature-icon"><i class="fa fa-check-square-o fa-5x" aria-hidden="true"></i></div>
+                    <p>Compliance with funding agency or journal publisher requirements for sharing data</p>
+                </div>
+
+                <div class="feature col-xs-12 col-sm-6">
+                    <div class="feature-icon"><i class="fa fa-code fa-5x" aria-hidden="true"></i></div>
+                    <p>A REST web service API and Python client for automated access</p>
+                </div>
+
+                <div class="feature col-xs-12 col-sm-6">
+                    <div class="feature-icon"><i class="fa fa-database fa-5x" aria-hidden="true"></i></div>
+                    <p>Discover and access data and models published by others</p>
+                </div>
+
+                <div class="feature col-xs-12 col-sm-6">
+                    <div class="feature-icon"><i class="fa fa-desktop fa-5x" aria-hidden="true"></i></div>
+                    <p>Web apps for visualizing, analyzing, and running models on data in HydroShare</p>
+                </div>
+            </div>
+        </div>
+    </section>
+
+    <section class="container">
         <h2 class="home-page-block-title">How it works</h2>
         <div class="row home-page-block">
-            <div class="steps-container">
-                <div class="col-xs-12 col-md-4 col-lg-2 col-lg-offset-1 step-wrapper slideInBlock">
-                    <div class="step-item">
-                        <div class="step-number">1</div>
-                        <h4>Create data</h4>
-                        <div class="description-container table-display">
-                            <span class="step-description table-cell-display">Collect your data using the same methods you use now. HydroShare supports a broad set of hydrologic data types.</span>
-                        </div>
-                        <div>
-                            <img class="img-responsive img-step" src="{{ STATIC_URL }}img/home-page/step1.png"
-                                 alt="Create data">
-                        </div>
+            <div class="steps-container col-xs-12">
+                <div class="step-item slideInBlock">
+                    <a href="https://help.hydroshare.org/hydroshare-resources/content-types/">
```

Comment:
Should these hydroshare help page links be set with `target="_blank"` ?

Reply:
Decided against this because the help pages are kind of considered to be part of the same site. And the UI is meant to reflect that.